t/plugin/limit-conn2.t

```diff
@@ -308,3 +308,134 @@ request latency is nil
 --- error_code: 400
 --- response_body
 {"error_msg":"failed to check the configuration of plugin limit-conn err: property \"rejected_msg\" validation failed: string too short, expected at least 1, got 0"}
+
+
+
+=== TEST 9: set key type to var_combination
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "plugins": {
+                            "limit-conn": {
+                                "conn": 2,
+                                "burst": 1,
+                                "default_conn_delay": 0.1,
+                                "rejected_code": 503,
+                                "key": "$http_a $http_b",
```

Comment:
Are the two variables separated by a space? Depending on the document it is separated by `|`.

Reply:
Thanks for your tips, two variables are separated by a space, doc updated.

Reply:
@tzssangglass  would you like to take a look at this PR again? doc updated.